app/src/main/java/io/github/droidkaigi/confsched/api/DroidKaigiClient.java

```diff
@@ -28,11 +30,13 @@
 
     private static final String END_POINT = "https://raw.githubusercontent.com";
     private static final String GOOGLE_FORM_END_POINT = "https://docs.google.com/forms/d/";
+    private static final String GITHUB_END_POINT = "https://api.github.com";
```

Comment:
I agree with you because this code pass few time.

If this code called many times, it is better to define static variables `GITHUB_END_POINT` because it does not create extra String instance. But it is not necessary here.